src/main/java/RandomComputerPlayer.java

```diff
@@ -0,0 +1,35 @@
+import java.util.List;
+import java.util.Random;
+
+public class RandomComputerPlayer implements Player{
+
+    private String marker = "";
+    private String name = "Computer";
+
+    public String getMarker(){
+        return marker;
+    }
+
+    public String getName(){
+        return name;
+    }
+
+    public void build(String marker) {
+        this.marker = marker;
+    }
+
+    public List<String> makeMove(Board currentBoard) {
+        int cellPosition = getCellPosition(0, 8);
```

Comment:
Why no call directly the method generateWithinRange? 

I would also call the method getMoveBetween(int min, int max). This way you communicate the intention of the method and you don't have to read the code to understand it 👍